internal/controller/services/monitoring/resources/prometheus-web-tls-ca-secret-job.tmpl.yaml

```diff
@@ -0,0 +1,106 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: prometheus-tls-ca-copy
+  namespace: {{.Namespace}}
+  labels:
+    platform.opendatahub.io/part-of: monitoring
+  ownerReferences:
+  - apiVersion: {{.Component.APIVersion}}
+    kind: {{.Component.Kind}}
+    name: {{.Component.Name}}
+    uid: {{.Component.UID}}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: prometheus-tls-ca-copy
+  namespace: {{.Namespace}}
+  labels:
+    platform.opendatahub.io/part-of: monitoring
+  ownerReferences:
+  - apiVersion: {{.Component.APIVersion}}
+    kind: {{.Component.Kind}}
+    name: {{.Component.Name}}
+    uid: {{.Component.UID}}
+rules:
+- apiGroups: [""]
+  resources: ["secrets"]
+  verbs: ["get", "create", "update", "patch"]
+  resourceNames: ["prometheus-web-tls-ca"]
+- apiGroups: [""]
+  resources: ["secrets"]
+  verbs: ["create"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: prometheus-tls-ca-copy
+  namespace: {{.Namespace}}
+  labels:
+    platform.opendatahub.io/part-of: monitoring
+  ownerReferences:
+  - apiVersion: {{.Component.APIVersion}}
+    kind: {{.Component.Kind}}
+    name: {{.Component.Name}}
+    uid: {{.Component.UID}}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: prometheus-tls-ca-copy
+subjects:
+- kind: ServiceAccount
+  name: prometheus-tls-ca-copy
+  namespace: {{.Namespace}}
+---
+# Job to copy CA from ConfigMap to Secret (MonitoringStack requires Secret)
```

Comment:
Sorry about the additional complexity. It would make sense to streamline this part and let the cluster observability operator rely on the service-CA operator to provision automatically all the TLS material.

Reply:
https://issues.redhat.com/browse/COO-1270